net.jammr.jammr.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The appdata, desktop file and icons should be part of the release tarball if possible.

Reply:
Understood.  This initial manifest is for the existing jammr 1.2.8 release so I can't add these files retrospectively.

I will merge the appdata, desktop file, and icons into wahjam.git in the next release and then they can be removed from the manifest.

Hope it's okay to include them here until the next jammr release.